app/routes/_layout+/assets.$assetId.qr.tsx

```diff
@@ -66,7 +81,12 @@ export default function QRPreview() {
           </h6>
         </div>
         <figure className="qr-code flex  justify-center">
-          <img src={data.qr.src} alt={`${data.qr.size}-shelf-qr-code.png`} />
+          <ImagePreview
+            ref={imagePreviewRef}
+            qr={data.qr.src}
+            size={data.qr.size}
+            logo="/images/shelf-symbol.png"
```

Comment:
This is good but once I checked it this image doesn't look that great because of the resizing. Can you please add this image to the assets and use it instead. https://cdn.discordapp.com/attachments/1043194358347862078/1143485720158294107/shelf-icon-28px.png